omod/src/main/webapp/fragments/extension.gsp

```diff
@@ -3,8 +3,20 @@
     config.require("contextModel")
 %>
 
+<%
```

Comment:
This generally looks good to me @ljoedavessenn .  Do the path and target variables need to be defined / initialized first?

Reply:
@mseaton No they don't need initialization. By default I consider http/https path as external, and any other are considered internal and follow the same logic as before. Also I open external url in another tab.
Maybe I could upgrade the code so that those parameters can be optionally passed in the configuration.